src/Sylius/Component/Currency/composer.json

```
@@ -53,7 +53,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Sylius\\Component\\Currency\\spec\\": "spec/"
+            "Tests\\Sylius\\Component\\Currency\\": "Tests/"
```

I would be for using lowercase for `tests` directory, especially in components/bundles where we are introducing this directory
```suggestion
            "Tests\\Sylius\\Component\\Currency\\": "tests/"
```